File: MCItalia-API/src/main/java/it/andrearossi/mcitaliaapi/crafting/CraftingList.java
package it.andrearossi.mcitaliaapi.crafting;

import it.andrearossi.mcitaliaapi.utils.html.JsonCategoryList;

public class CraftingList extends JsonCategoryList<Crafting, CraftingCategory> {

	public CraftingList(String status, Crafting[] items, CraftingCategory[] categories) {
		super(status, items, categories);
	}
}

File: src/main/java/it/andrearossi/mcitaliaapi/crafting/CraftingList.kt
package it.andrearossi.mcitaliaapi.crafting

import it.andrearossi.mcitaliaapi.utils.html.JsonCategoryList

class CraftingList(status: String?, items: Array<Crafting?>?, categories: Array<CraftingCategory?>?) : JsonCategoryList<Crafting?, CraftingCategory?>(status!!, items!!, categories!!) 